### PR TITLE
Upgrade to ghc-lib-0.20190531

### PR DESCRIPTION
**Pull request by shayne-fletcher-da**
This PR upgrades DAML to `ghc-lib-0.20190531` (ghc `495a65cbc48d5209f30fd4248fc11ab06b05d4c3`).

### CODE REVIEW

File: compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs [hunk 1]
```diff
@@ -62,25 +63,68 @@ runGhcEnv env act = do
...
+          Platform{
+            platformWordSize=8
+          , platformOS=OSUnknown
+          , platformUnregisterised=True
+        }
+        platformConstants =
+          PlatformConstants {
+            pc_DYNAMIC_BY_DEFAULT=False
+          , pc_WORD_SIZE=8
+          -- , pc_STD_HDR_SIZE=1
```
Comment (neil-da) [inline compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs]: Can we just delete these rather than commenting out

Comment (shayne-fletcher-da) [reply compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs]: Oops!

Comment (shayne-fletcher-da) [reply compiler/haskell-ide-core/src/Development/IDE/UtilGHC.hs]: ✔️

**Comment by shayne-fletcher-da** [comment]
Something is very, very wrong with regarding this PR in CI. I'm going to close it and open another.

**Pull request closed without merging**